ecosystem/resources/members/mqt-qmap.toml

```
@@ -0,0 +1,14 @@
+name = "MQT QMAP"
+url = "https://github.com/cda-tum/mqt-qmap"
+description = "A tool for quantum circuit compilation for various qubit technologies developed as part of the Munich Quantum Toolkit (MQT)."
+licence = "MIT license"
+contact_info = "quantum.cda@xcit.tum.de"
+alternatives = "Any other layout and routing techniques for superconducting systems, Clifford synthesis approach, or compilation tool for neutral atom quantum computers."
+affiliations = "Technical University of Munich (TUM), Germany"
+labels = [ "Transpiler plugin", "Paper implementation",]
```

The `Transpiler plugin` label is supposed to be for projects that implement Qiskit's [transpiler plugin interface](https://docs.quantum.ibm.com/guides/create-transpiler-plugin). I'm gonna remove it, but we can add it back if you do decide to implement that interface in the future.

```suggestion
labels = [ "Paper implementation",]
```